### PR TITLE
webfs: use sha256 and fix audit

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by xu-cheng** [comment]
@nijikon Can you fix all the audit problems revealed by `brew audit --strict`?

File: Library/Formula/webfs.rb [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
   end
 
   def install
     ENV["prefix"]=prefix
-    system "make install mimefile=/etc/apache2/mime.types"
+    system "make", "install", "mimefile=/etc/apache2/mime.types"
+  end
+
+  test do
+    system "webfsd", "-h"
```
Comment (bfontaine) [inline Library/Formula/webfs.rb]: Please use `"#{bin}/webfsd"`.

**Comment by nijikon** [comment]
@xu-cheng @bfontaine please review I'm keen for more feedback :dancer:

File: Library/Formula/webfs.rb [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
   end
 
   def install
     ENV["prefix"]=prefix
-    system "make install mimefile=/etc/apache2/mime.types"
+    system "make", "install", "mimefile=/etc/apache2/mime.types"
+  end
+
+  test do
+    assert_match %r{^This is a lightweight http server for static content}, shell_output("#{bin}/webfsd -h 2>&1", 1)
```
Comment (xu-cheng) [inline Library/Formula/webfs.rb]: Can we do something more substantial? You can see `serveit.rb` as an example. Thanks!

**Comment by mikemcquaid** [comment]
Makes sense to me :+1:

**Comment by DomT4** [comment]
If this wasn't using OpenSSL before and it needs to, we need to add a revision line here :trollface:.

File: Library/Formula/webfs.rb [hunk 3]
```diff
@@ -1,18 +1,29 @@
-require 'formula'
-
 class Webfs < Formula
   desc "HTTP server for purely static content"
-  homepage 'http://linux.bytesex.org/misc/webfs.html'
-  url 'http://dl.bytesex.org/releases/webfs/webfs-1.21.tar.gz'
-  sha1 'a38880d8cb21e415244d220115ede7b573ac890c'
+  homepage "http://linux.bytesex.org/misc/webfs.html"
+  url "http://dl.bytesex.org/releases/webfs/webfs-1.21.tar.gz"
```
Comment (vszakats) [inline Library/Formula/webfs.rb]: This URL would better with `https://`.

**Comment by nijikon** [comment]
There is a problem, the fork code does not kill the proper pid. Pid that it wants to kill is 83819 but the actual pid for webfsd is 83820. Any idea how to handle this?

```
balrog:local n$ brew test webfs
/usr/local/Library/brew.rb (Formulary::FormulaLoader): loading /usr/local/Library/Formula/webfs.rb
Testing webfs
/usr/bin/sandbox-exec -f /tmp/homebrew20150724-83808-1rlfd4h.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Formula/webfs.rb
/usr/local/Library/Homebrew/test.rb (Formulary::FromPathLoader): loading /usr/local/Library/Formula/webfs.rb
83819
==> curl localhost:8000
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   473  100   473    0     0  71106      0 --:--:-- --:--:-- --:--:-- 78833
83819
syslog -F '$((Time)(local)) $(Sender)[$(PID)]: $Message' -k Time ge 1437715751 -k Sender kernel -o -k Time ge 1437715751 -k Sender sandboxd | grep deny > /Users/n/Library/Logs/Homebrew/webfs/sandbox.test.log
balrog:local n$ ps aux|grep webf
n               83832   0.0  0.0  2432772    648 s027  S+    7:29AM   0:00.00 grep webf
n               83820   0.0  0.0  2454564    468   ??  Ss    7:29AM   0:00.00 /usr/local/Cellar/webfs/1.21_1/bin/webfsd
```

**Comment by mikemcquaid** [comment]
@nijikon Perhaps it's daemonizing and needs to be told not to do that?

File: Library/Formula/webfs.rb (hunk 3 above)
Comment (mikemcquaid) [inline Library/Formula/webfs.rb]: :+1:

**Comment by nijikon** [comment]
@mikemcquaid seems legit now.

File: Library/Formula/webfs.rb [hunk 4]
```diff
@@ -1,18 +1,30 @@
...
 
   def install
     ENV["prefix"]=prefix
-    system "make install mimefile=/etc/apache2/mime.types"
+    system "make", "install", "mimefile=/etc/apache2/mime.types"
+  end
+
+  test do
+    begin
+      pid = fork { exec bin/"webfsd -F" }
```
Comment (xu-cheng) [inline Library/Formula/webfs.rb]: A small nit: it should be `exec bin/"webfsd", "-F"`. I will fix this when merging.

**Comment by xu-cheng** [comment]
Thanks @nijikon

**Pull request closed without merging**